protocol/0001-MKTF-market_framework.md

```diff
@@ -32,6 +32,7 @@ Data:
 - **Position Decimal Places**: number of decimal places for orders and positions, i.e. if this is 2 then the smallest increment that can be traded is 0.01, for example 0.01 BTC in a `BTSUSD` market.
   - If this is negative e.g. -3 this means that the smallest order and position is of size 1000.
   - Accepted values are `-6,...,-1,0,1,2,...,6`.
+- **Tick size**: the minimum change in quote price for the market. Order prices and offsets for pegged orders must be given as an exact multiple of the tick size. For example if the tick size is 0.02 USD. then a price of 100.02 USD is acceptable and a price of 100.03 USD is not. The tick size of a market can be updated through governance.
```

Comment:
Would it be clearer/cleaner if tick size were specified *in* market price decimals? e.g. for the above example it would just be 2. Would enforce that tick size is always compatible with the market decimals and possibly make the maths easier.

Reply:
That was definitely the intention when writing but reading back I realise that is not clear so thanks for flagging!